codespell_lib/data/dictionary.txt

```diff
@@ -8138,6 +8138,12 @@ debufs->debugfs
 debugee->debuggee
 debuger->debugger
 debugg->debug
+debuggg->debug
```

Comment:
Please can you add `debugggs->debugs` too.

Reply:
→ 8b54cf7f6a6b4e1e3a2010e551604982e9c029b5